### ISSUE TITLE
Change -dump output names in parallel

### ISSUE THREAD

**Issue by aowen87** [aowen87/FOO#290]
Tried to compare dumps from serial run and parallel run.
Serial run had names that began: 0000.input.avtblahblah 0001.output.avtblahblahblah
Whereas the parallel names:  0000.0.input.abtblahbblah 0000.1.output.avtblahblahblah
Looks like "processor.dumpnumber." But there were no dumps for any processor other than proc 0.

If it's alway processor 0 that is doing the dumping, can the parallel dumps have names consistent with the serial naming scheme, for easier comparison?  I had to enter each file pair individually on the command line rather than allowing xxdiff to read the whole directory.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Kathleen Biagas
Original creation: 04/25/2012 12:14 pm
Original update: 05/04/2012 02:44 pm
Ticket number: 1034
